modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/resources/META-INF/resources/mfa_fido2_checker/verify_browser.jsp

```diff
@@ -18,65 +18,24 @@
 
 <%
 String assertionReqest = (String)request.getAttribute("assertionReqest");
+
+Map<String, Object> additionalProps = HashMapBuilder.<String, Object>put(
+	"assertionReqest", assertionReqest
```

Comment:
I tried to run verification, but the popup dialog didnt show up, during debuging, I noticed AuthenticationTransformer is returned because undefined assertionRequest, and it turns of here are three typos in the word of **assertionReqest** vs **assertionRequest**

Reply:
Oh, looks like I had the typo in server code as well. The reason why it worked before was that I had the typo everywhere. :smile: 
Lets correct the typo to assertionRequest.

Reply:
I'll fix the typo :+1: